docs/user-manual/en/web-server.md

```diff
@@ -89,3 +89,19 @@ Here is an example configuration:
    <request-log filename="${artemis.instance}/log/http-access-yyyy_MM_dd.log" append="true" extended="true"/>
 </web>
 ```
+
+## Proxy Forwarding
+
+The proxy and load balancers usually supports `X-Forwarded` headers
```

Comment:
… usually _support_ …

Reply:
thanks, I have just pushed a commit to fix the typo